x-pack/plugins/asset_inventory/server/plugin.ts

```diff
@@ -0,0 +1,54 @@
+import type {
+  PluginInitializerContext,
+  CoreSetup,
+  CoreStart,
+  Plugin,
+  Logger,
+} from '@kbn/core/server';
+
+import type { AssetInventoryPluginSetup, AssetInventoryPluginStart } from './types';
+import { defineRoutes } from './routes';
+import { initializeTransforms } from './create_transforms/create_transforms';
+
+export class AssetInventoryPlugin
+  implements Plugin<AssetInventoryPluginSetup, AssetInventoryPluginStart>
+{
+  private readonly logger: Logger;
+
+  #isInitialized: boolean = false;
```

Comment:
:opinion nit

`#isInitialized` in the CSPM plugin also,  this short hand for a private variable is odd IMO.

Is it possible to...

```suggestion
   private isInitialized: boolean = false;
```